apps/calendar/js/controllers/notifications.js

```diff
@@ -55,26 +55,27 @@ function issueNotification(alarm) {
     }
 
     var begins = calc.dateFromTransport(busytime.start);
-    var distance = dateFormat.fromNow(begins);
-    var now = new Date();
+    return dateFormat.relativeDate(begins).then(distance => {
```

Comment:
How is it different from resolving a Promise here? I'm not sure how to change my code to use `co` here.